MIGRATION.md

```diff
@@ -230,9 +233,27 @@ This was done to remove a hard dependency on `https-proxy-agent` when running
    authorization: 'authorization'
  }`
 
-### 8. Bug fix: Setting an explicit `algorithm` when generating a wallet works now
+### 8. `Wallet` functions default to `ed25519` instead of `secp256k1` signing algorithm
+
+With 3.0, we updated the default signing algorithm used by the Wallet object from secp256k1 to ed25519 in order to default to the higher-performance algorithm. This is a breaking change to all functions used to generate a Wallet, so if you have a pre-existing XRPL account that you're using to generate a specific Wallet using older versions of xrpl.js, you must specify that you are using secp256k1 as the algorithm to decode your private key / seed / etc to get the same behavior as before. See below for specifically how to update your code.
+
+If you are creating new accounts each time (ex. via `Client.fundWallet`), you do not need to specify the signing algorithm.
+
+**Before**
+
+`Wallet.fromSeed('s...')`
+`Wallet.generate()`
+`Wallet.fromEntropy(entropy)`
+`deriveKeyPair(seed="s...")`
+`generateSeed()`
+
+**After**
 
-`Wallet.generate()` and `Wallet.fromSeed` were ignoring the `algorithm` parameter. This means that if you were manually specifying `algorithm` in any `Wallet` constructors, you may generate a different `Wallet` keypair when upgrading to 3.0. In that case to get the same generated wallets as before, don’t specify the `algorithm` parameter.
+`Wallet.fromSeed(seed='s...',algorithm: 'ecdsa-secp256k1')`
+`Wallet.generate(algorithm: 'ecdsa-secp256k1')`
+`Wallet.fromEntropy(entropy, opts={algorithm: 'ecdsa-secp256k1'})`
+`deriveKeypair(seed='s...', opts={ algorithm: 'ecdsa-secp256k1' }) (ripple-keypairs)`
+`generateSeed({ entropy, algorithm: 'ecdsa-secp256k1' } (ripple-keypairs)`
```

Comment:
ditto here

Reply:
fixed in [e216200](https://github.com/XRPLF/xrpl.js/pull/2659/commits/e216200ec2a94a39939dba55039bc5bedb389ed7)